prog/kubernetes/kubernetes_cluster_nexus.rb

```diff
@@ -90,8 +90,18 @@ def before_run
     donate
   end
 
+  label def sync_kubernetes_services
```

Comment:
nit: this label can be placed below `wait` to match order of reading with order of execution.